set-vars.md

```diff
@@ -0,0 +1,155 @@
+| `time zone`                     | The default time zone for the current session. See [`SET TIME ZONE` notes](#set-time-zone) below. | "`UTC`"                                                                                                                                                                     | Yes
+| `default_transaction_isolation` | The default transaction isolation level for the current session. See [Transaction parameters](transactions.html#transaction-parameters) and [`SET TRANSACTION`](set-transaction.html) for more details. | Settings in connection string, or "`SERIALIZABLE`" if not specified.  | Yes
+| `client_encoding`               | Ignored; recognized for compatibility with PostgreSQL clients. Only possible value is "`UTF8`". | N/A                                                                                                                                                                           | No
+| `client_min_messages`           | Ignored; recognized for compatibility with PostgreSQL clients. Only possible value is "`on`". | N/A                                                                                                                                                                             | No
+| `standard_conforming_strings`   | Ignored; recognized for compatibility with PostgreSQL clients. | N/A                                                                                                                                                                                                            | No
+| `extra_float_digits`            | Ignored; recognized for compatibility with PostgreSQL clients. | N/A                                                                                                                                                                                                            | No
+
+## Examples
+
+### Set Simple Variables
+
+The following demonstrates how `SET` can be used to configure the
+default database for the current session:
+
+~~~ sql
+> SET database = bank;
+> SHOW database;
+~~~
+
+~~~
++----------+
+| database |
++----------+
+| bank     |
++----------+
+(1 row)
+~~~
+
+### Set Variables To Values Containing Spaces
+
+The following demonstrates how to use quoting to use values containing spaces:
+
+~~~ sql
+> SET database = "database name with spaces";
+> SHOW database;
+~~~
+
+~~~
++---------------------------+
+|         database          |
++---------------------------+
+| database name with spaces |
++---------------------------+
+(1 row)
+~~~
+
+### Set Variables To A List Of Values
+
+The following demonstrates how to assign a list of values, which
+is e.g. relevant for `search_path`:
```

Comment:
Is using a list applicable to anything but `search_path`? I supposed additional such variable may be added in the future...

In any case, I might just cut everything after the comma.